pages/levels/5.vue

```diff
@@ -1,12 +1,12 @@
 <script lang="ts">
 import { defineComponent } from "vue";
 
-const code = `    `;
+const code = `    <div @click="switchButtonColor">Click me!</div>`;
```

Comment:
Eine Sache die mir noch eingefallen ist: die Syntax mit `@click` ist Vue-spezifisch und schaut bei anderen Frameworks anders aus.
```suggestion
const code = `    <div onclick="switchButtonColor()">Click me!</div>`;
```